.env.example

```diff
@@ -0,0 +1,4 @@
+CLIENT_ID=
+CLIENT_SECRET=
+CONFIG_FILE=config.json
```

Comment:
If we're going to have environment variables, I would prefer to have only one source of configuration, because this is supposed to be a small simple app, so we can remove the `config.json` entirely, and have all configuration variables in `.env` instead.

Reply:
Done!